packages/toolpack/src/webpack/config/base.ts

```diff
@@ -150,6 +157,22 @@ export function baseWebpackChain({
           : CssMinimizerPlugin.cssnanoMinify
       }
     ]);
+
+    if (analyze) {
+      const targetName = getSimplifiedTargetName(name);
+      config
+        .plugin('private/bundle-analyzer-plugin')
+        .use(BundleAnalyzerPlugin, [
+          {
+            logLevel: 'info',
```

Comment:
Using Webpack Bundle Analyzer' s log.
Otherwise, we need a shuvi plugin to get real reportFilename and statsFilename (we cannot know target)